pages/docs/mutation.en-US.md

```diff
@@ -239,3 +247,120 @@ function Profile () {
   )
 }
 ```
+
+## useSWRMutation
+
+SWR also provides `useSWRMutation` as a hook for remote mutations. The remote mutations are only triggered manually, instead of automatically like `useSWR`.
+
+```jsx
+import useSWRMutation from 'swr/mutation'
+
+async function getData(url, { arg: token }) {
```

Comment:
Giving less changing of the original api signature in the first example to let users remember it easily
```suggestion
async function getData(url, { arg }) {
```